src/InTextAnnotationParser.php

```diff
@@ -141,6 +141,64 @@ public function parse( &$text ) {
 	}
 
 	/**
+	 * @since 2.4
+	 *
+	 * @param string $text
+	 *
+	 * @return text
+	 */
+	public static function obscureAnnotation( $text ) {
+
+		// Convert possible internal URL encoding
+		$text = str_replace( array( '%5B', '%5D' ), array( '[', ']' ), $text );
```

Comment:
You don't need this local var, or the override the previous value of the parameter. You can just inline this below. At the same time it seems like a good candidate for extraction into its own method, due to the comment and the presence of low level details.